apps/playground-web/shared/utils/shellUtils.ts

```diff
@@ -14,7 +14,14 @@ export const handleCommand = async ({ command, setOutput }: CommandHandler) => {
   }
   try {
     result = await executeShellCommandOnServer(cmd, args);
-    setOutput((prevOutput) => [...prevOutput, newOutput, result]);
+    if (result?.body?.data) {
+      setOutput((prevOutput) => [...prevOutput, newOutput, result?.body?.data]);
+    } else if (result?.body?.error) {
+      setOutput((prevOutput) => [...prevOutput, newOutput, result?.body?.error]);
+    } 
+    const commandsLeft = result?.headers?.['x-ratelimit-remaining'];
+    const cleanupTimeLeft = 10;
```

Comment:
Same as above

Reply:
yes, I have to fix this. waiting on this issue https://github.com/DiceDB/playground-mono/issues/32